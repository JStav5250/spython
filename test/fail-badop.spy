int x = 3

float z = 4.0

int y = z + 3
print(y)
