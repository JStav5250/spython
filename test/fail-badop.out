TypeError: unsupported operand type(s) for binary +: 'float' and 'int'!!
